scripts/release

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env ruby
+
+require "date"
+require "optparse"
+
+def run(command, abort_on_fail: true)
+  puts "> #{command}" if $VERBOSE
+
+  system(command, exception: abort_on_fail)
+rescue => e
+  raise "#{command.inspect} failed: #{e}"
+end
+
+channel = ENV.fetch("FOREM_RELEASE_CHANNEL", "stable")
+date = ENV.fetch("FOREM_RELEASE_DATE", Date.today.to_s).gsub(/-/, "")
```

Comment:
I'm surprised Rubocop didn't suggest this but `delete` is often preferred over substituting an empty string.

```suggestion
date = ENV.fetch("FOREM_RELEASE_DATE", Date.today.to_s)..delete("-")
```

Reply:
Oh, interesting, TIL. I've always used `gsub` because I thought `String#delete` returned what was deleted like `Array#delete` does.

```ruby
[3] pry(main)> Date.today.to_s.delete'-'
=> "20210513"
[4] pry(main)> [1,2,3].delete 1
=> 1
```

> I'm surprised Rubocop didn't suggest this

It's probably because it's not a `.rb` file.